src/screens/ActivitiesPage.js

```diff
@@ -14,6 +14,9 @@ const ActivitiesPage = () => {
     const { userActivityCount } =
         useFirebaseDB();
     const [user] = useAuthState(firebaseAuth);
+    const monthGoal = 5;
+    const metMonthGoal = userActivityCount >= monthGoal;
+    const progressIncrement = 100 / monthGoal;
```

Comment:
Love the effort to move towards more interchangeability but this causes a "reverse" progress bar when a user goes over the monthly goal. Let's go back to the *20 
<img width="1680" alt="Screen Shot 2022-04-20 at 12 47 49 AM" src="https://user-images.githubusercontent.com/43453743/164152628-7737c92c-518a-4dd9-9bb4-56dd8b94df31.png">